.github/workflows/publish.yml

```diff
@@ -1,4 +1,4 @@
-name: Publish to master
+name: Publish Jobs
```

Comment:
I actually find "Publish to master" more eloquent / clear

Reply:
it's the "Publishing Workflow" - not a specific job